backend/src/packages/users/user.service.ts

```
@@ -95,8 +95,15 @@ class UserService implements Service {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Service['delete']> {
-    return Promise.resolve(true);
+  public async delete(id: number): Promise<boolean> {
+    const user = await this.findById(id);
+    if (!user) {
+      return false;
+    }
```

we should throw an error in case there is no user with this id